### ISSUE TITLE
No keywords extracted. 

### ISSUE THREAD

**Issue by BrandonJamesBorders** [venveo/craft-documentsearch#1]
Hello! Evaluating the plugin but I'm not having any luck getting keywords to populate. 

![image](https://user-images.githubusercontent.com/6044898/68511392-dbaf8900-0243-11ea-9fb2-f5767677b19e.png)

![image](https://user-images.githubusercontent.com/6044898/68511416-ee29c280-0243-11ea-8bf0-4bc2016e9d3c.png)

![image](https://user-images.githubusercontent.com/6044898/68511438-f84bc100-0243-11ea-94a5-a48509a3a616.png)

searchindex db:
![image](https://user-images.githubusercontent.com/6044898/68511508-1f09f780-0244-11ea-9e12-a3e76064fe70.png)

If I run the command in my terminal, I do get text out of the document, so I'm not sure what is going on. Not seeing any logging going on as a result of your Craft::info calls either.

**Comment by Mosnar** [comment]
Hey! Thanks for reporting, sorry you're having trouble. Would you mind sharing your sample PDF so we can test it out on our end?

You can email it to me via ransom@venveo.com

Could you also please share your Craft version so I can make sure I test on the same version?

**Comment by BrandonJamesBorders** [comment]
Funny enough, or not, I used this one to test initially. 
http://www.africau.edu/images/default/sample.pdf

Craft 3.3.15

**Comment by Mosnar** [comment]
@BrandonJamesBorders I was able to reproduce the issue and will have a fix rolled out ASAP (hopefully today). 

I haven't looked into it much yet, but my hunch is that this is a regression caused by a Craft update.

**Comment by Mosnar** [comment]
@BrandonJamesBorders Okay, so it turns out this this is actually a different issue than I thought. 

I ran the plugin on the Craft 3.4 Alpha and ran into issues due to the delta-changes feature - so we'll need to address that.

However, after changing to Craft 3.3.15, I found that the plugin was working as expected. I reviewed your ticket and realized the Volume checkboxes in your screenshot doesn't actually have anything besides "All", meaning the plugin isn't actually doing anything. Upon investigation, I realized we're not showing all available volumes, only public volumes. So, I'm assuming your PDF is located in a volume that doesn't have URLs and you have no other public volumes.

I just released 1.0.2 that addresses this. After you update, make sure you go into the plugin settings and check and re-save the settings to include the appropriate volume.

Keep me posted!
